### PR TITLE
Release 10.9.2

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-9-2.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: "WaveMaker 10.9.2 - Release date: 01 November 2021"
+id: ""
+sidebar_label: "Release v10.9.2"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Bug Fixes
+
+- Fixed an issue related to the setFilter method of the CRUD variable showing console errors.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-9-2.md:10]: Please mention the following bugg fixes as well:
https://wavemaker.atlassian.net/browse/WMS-21289
https://wavemaker.atlassian.net/browse/WMS-21344
https://wavemaker.atlassian.net/browse/WMS-20568

**Review by vibhus — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-9-2.md (hunk 1 above)
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-9-2.md:11]: Done. Added 21289, 21344, 20568 tickets.

File: learn/wavemaker-release-notes/v10-9-2.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+sidebar_label: "Release v10.9.2"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Bug Fixes
+
+- The improved search capability of the wizard widget in the widget panel; added new keywords, including step, stepper, and wizard. 
+- Fixed an issue related to the setFilter method of the CRUD variable showing console errors.
+- Fixed an issue related to the date picker inside a dialog that did not show correctly.
+- Fixed an issue related to the generated-angular-app folder in WaveMaker Studio.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-9-2.md:13]: Please change this to:
"The generated Angular app can now be viewed in studio while working on the project."

**Review by vibhus — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-9-2.md (hunk 2 above)
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-9-2.md]: Done.

**Review by vibhus — APPROVED** [review]

**Pull request merged**